src/main/java/io/deepstream/DeepstreamClient.java

```diff
@@ -289,6 +289,14 @@ public ConnectionState getConnectionState() {
     }
 
     /**
+     * Set global connectivity state.
```

Comment:
can we add a slight amount more documentation here to mention why this would be used? (We'll use it for the documentation on the website)